### PR TITLE
Adds a cluster comment for tests produced by Smart Fuzzer

### LINKED ISSUE #430: Summaries should handle correctly UtExecution generated by Fuzzer
**Description**

I propose to make unified handling for all tests, generated by Fuzzer and others

**Expected behavior**

We should make clusterization for Fuzzing tests too and have the same names for Fuzzer and other produced by summarization framework

Below you could see that cluster for tests generated by Fuzzer is empty.

Empty cluster for fuzzer

### PR DESCRIPTION
**Pull request by amandelpie**
# Description

Adds a cluster comment for tests produced by Smart Fuzzer

Fixes #430 

## Type of Change

- New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: utbot-summary/src/main/kotlin/org/utbot/summary/Summarization.kt [hunk 1]
```diff
@@ -97,14 +106,39 @@ class Summarization(val sourceFile: File?, val invokeDescriptions: List<InvokeDe
...
+                clustersToReturn.add(
+                    UtExecutionCluster(
+                        UtClusterInfo(clusterHeader, null),
+                        successfulFuzzerExecutions
+                    )
                 )
-            )
+            }
+
+            if(unsuccessfulFuzzerExecutions.isNotEmpty()) {
```
Comment (onewhl) [inline utbot-summary/src/main/kotlin/org/utbot/summary/Summarization.kt:132]: please, add a space after `if`

File: utbot-summary/src/main/kotlin/org/utbot/summary/TagGenerator.kt [hunk 2]
```diff
@@ -148,13 +150,13 @@ enum class ClusterKind {
...
 private fun UtExecutionResult.clusterKind() = when (this) {
-    is UtExecutionSuccess -> ClusterKind.SUCCESSFUL_EXECUTIONS
-    is UtImplicitlyThrownException -> if (this.exception.isCheckedException) ClusterKind.CHECKED_EXCEPTIONS else ClusterKind.ERROR_SUITE
-    is UtExplicitlyThrownException -> if (this.exception.isCheckedException) ClusterKind.CHECKED_EXCEPTIONS else ClusterKind.EXPLICITLY_THROWN_UNCHECKED_EXCEPTIONS
-    is UtOverflowFailure -> ClusterKind.OVERFLOWS
-    is UtTimeoutException -> ClusterKind.TIMEOUTS
-    is UtConcreteExecutionFailure -> ClusterKind.CRASH_SUITE
-    is UtSandboxFailure -> ClusterKind.SECURITY
+    is UtExecutionSuccess -> ExecutionGroup.SUCCESSFUL_EXECUTIONS
+    is UtImplicitlyThrownException -> if (this.exception.isCheckedException) ExecutionGroup.CHECKED_EXCEPTIONS else ExecutionGroup.ERROR_SUITE
```
Comment (onewhl) [inline utbot-summary/src/main/kotlin/org/utbot/summary/TagGenerator.kt:154]: would it look better if we add brackets?
```
if (...) {
...
} else {
...
}
```

**Review by onewhl — COMMENTED** [review]
Thank you! Please, take a look at several minor comments

File: utbot-summary-tests/src/test/kotlin/math/SummaryOfMathTest.kt [hunk 3]
```diff
@@ -219,10 +219,10 @@ class SummaryOfMathTest : SummaryTestCaseGeneratorTest(
         )
 
         val clusterInfo = listOf(
-            Pair(UtClusterInfo("SUCCESSFUL EXECUTIONS #0 for method ofDoubles(double[])", null), 3),
```
[the remark this answers is not in the document]
Comment (amandelpie) [reply utbot-summary-tests/src/test/kotlin/math/SummaryOfMathTest.kt:222]: what kind of builder do you mean?

**Review by onewhl — APPROVED** [review]

**Pull request merged**